frodo.js

```diff
@@ -1,11 +1,11 @@
 /**
```

Comment:
SCRIPT-This logic might be clearer if you used a switch statement instead of multiple if-else statements.